app/constants/screens.ts

```diff
@@ -70,6 +70,8 @@ export const THREAD = 'Thread';
 export const THREAD_FOLLOW_BUTTON = 'ThreadFollowButton';
 export const THREAD_OPTIONS = 'ThreadOptions';
 export const USER_PROFILE = 'UserProfile';
+export const CONVERT_GM_TO_CHANNEL = 'ConvertGMToChannel';
+export const TEAM_SELECTOR_LIST = 'TeamSelectorList';
```

Comment:
Same thing about alphabetic order here. I know it would be better to have this automatically fixed by a linter rule, but we don't have such rule right now. If you find something we can use for these particular cases, it is more than welcomed :smile:

Reply:
Done.